api/app/views/mno_enterprise/jpi/v1/admin/subscriptions/_subscription.json.jbuilder

```diff
@@ -9,6 +9,7 @@ json.available_licenses subscription.available_licenses
 json.external_id subscription.external_id
 json.custom_data subscription.custom_data
 json.provisioning_data subscription.provisioning_data
+json.available_edit_actions subscription.available_edit_actions
```

Comment:
Before we forget, this has been renamed to `available_actions`
https://github.com/maestrano/maestrano-hub/pull/972/files#diff-c8fbb65db37a91bbb5be88241f81e874R24

Reply:
@BrunoChauvet Oops, yeah I had changed it but not pushed up the changes.